modules/Movecraft/src/main/java/net/countercraft/movecraft/Movecraft.java

```diff
@@ -379,7 +381,19 @@ public void onEnable() {
 
             getServer().getPluginManager().registerEvents(new InteractListener(), this);
             if (worldEditPlugin != null) {
-                getServer().getPluginManager().registerEvents(new WorldEditInteractListener(), this);
+                final Class clazz;
+                try {
+                    clazz = Class.forName("net.countercraft.movecraft.compat.we6.IMovecraftRepair");
```

Comment:
As far as I can tell, this is pointless and could be simplified greatly

Reply:
well, I did it like this with possible future 1.13+ compatibility in mind due to that WE 7.0.0 for 1.13+ have quite different code than WE 6.1.9 that works up to 1.12.2. This can, however, be re-implemented if 1.13+ compat is implemented here

Reply:
Hm... Maybe we should implement 1.13 before continuing this pr then?